documentation/asciidoc/topics/ref_upgrades.adoc

```diff
@@ -26,3 +26,25 @@ ifdef::downstream[]
 If you upgrade {brandname} clusters manually and have upgraded the channel for your {ispn_operator} subscription from **8.1.x** to **8.2.x** you should apply the upgrade for the latest {brandname} 8.2.x version as soon as possible to avoid potential data loss that can result from an issue in 8.2.0.
 For more information, see link:{relnotes_openshift}[{ispn_operator} {ProductVersion} Release Notes].
 endif::downstream[]
+
+[discrete]
+== Upgrade types
+//Community Only
```

Comment:
@gustavonalle Should all of this be community only, including the 2nd level header?

I think it would be worth putting this into a standalone file `ref_upgrade_types.adoc` then you can wrap the includes directive in the assembly like so:

```
ifdef::community[]
include::{topics}/ref_upgrade_types.adoc[]
endif::community[]
```